README.md

```diff
@@ -115,6 +118,7 @@ Talks and presentations about diversity, consensual software and safe communitie
 
 - [9 Ways to Stop Hurting and Start Helping Women in Tech](https://www.youtube.com/watch?v=pUVhF3jDG08) - By Brianna Wu.
 - [Consensual Software: Prioritizing Trust & Safety for new features](https://www.youtube.com/watch?v=Ccw3VfE3P4M) - By Danielle Leong.
+- [Cathegory Theory in Life](https://youtu.be/ho7oagHeqNc) - By Eugenia Cheng
```

Comment:
I think we have a typo here, right? (`Category Theory in Life`) We must fix the typo before merging the PR 😄

Reply:
You are right! (I have a personal issue with the typos 😅)